requirements.txt

```diff
@@ -1,4 +1,5 @@
 firecloud==0.16.25
 pip==19.3.1
 tenacity==5.1.1
-pandas==0.25.3
+pandas==1.1.4
```

Comment:
I'm not sure if an older version of pandas was being used deliberately, but I found it was also broken when running a fresh build (again, probably missed because of layer caching).  updating the version fixed things.